README.md

```diff
@@ -445,6 +446,35 @@ All PRs will run [super-linter](https://github.com/github/super-linter). You can
 You can run [`terraform fmt --recursive`](https://www.terraform.io/docs/commands/fmt.html) to rewrite your terraform config-files to a [canonical format](https://www.terraform.io/docs/configuration/style.html).
 > :warning: [Terraform binary](https://www.terraform.io/downloads.html) must be available to do this.
 
+#### How to sync module with the latest template
+
+1 Add remote upstream to template origin
+```text
+git remote add template https://github.com/fredrikhgrelland/vagrant-hashistack-template.git
+```
+2 Fetch all
+```text
+git fetch --all
+```
+3 Checkout template master branch and pull. Its important to have it locally
+```text
+git checkout -b template-master template/master
+git pull
+```
+4 Checkout new branch from origin master of current module
+```text
+git checkout -b sync master
+git pull
```

Comment:
what is this pull here for? Do lines 466,467 do the same job as the below :
```
git checkout master
git pull
git checkout -b sync
```

Reply:
actually no. I think your version is correct